src/utils/SVGIcons.ts

```diff
@@ -0,0 +1,24 @@
+import { $$, Dom } from './Dom';
+
+declare var require: (svgPath: string) => string;
+
+export class SVGIcons {
+  public static search = require('svg/search');
+  public static more = require('svg/more');
+  public static facetLoading = require('svg/facet-loading');
+  public static checkboxMoreValues = require('svg/checkbox-hook-exclusion-more');
+  public static arrowUp = require('svg/arrow-up');
+  public static arrowDown = require('svg/arrow-down');
+
+  private static parser = new DOMParser();
+
+  public static addClassToSVGInContainer(svgContainer: HTMLElement, classToAdd: string) {
```

Comment:
If we need something like that, I would simply add something generic to the `Dom` class.

`$$(svgContainer).addClassTo('svg' , classToAdd)`, which use the first parameter as a selector, and `$$.addClass` which already exists in `Dom`